cluster/gce/config-common.sh

```diff
@@ -98,6 +98,14 @@ function get-cluster-ip-range {
   echo "${suggested_range}"
 }
 
+function get-alias-range-size() {
+  for pow in {0..8}; do
+    if (( 1 << $pow > $1 * 2 )); then
+      echo $((32 - pow))
```

Comment:
why use echo as the return style? Isn't the normal mechanism to set a VARIABLE and then use that?

Reply:
I am just following the code style in the file cluster/gce/config-common.sh. Also returning with echo can eliminate unnecessary variable existance assumptions.